commonknowledge/join
# feat: add mailchimp integration

## Description
Adds settings and code necessary to sign up users to Mailchimp.

## Motivation and Context
https://linear.app/commonknowledge/issue/CK-3661/integration-with-mailchimp

## How Can It Be Tested?
1. Get a Mailchimp API key and Audience ID
2. Add these to the Join Flow settings page
3. Enable Mailchimp in the Features tab
4. Sign up with Minimal Form or Normal Form
5. Check email has been added to Mailchimp audience

## How Will This Be Deployed?
Normal deployment, need to add credentials.
